app/models/attachment.js

```diff
@@ -219,9 +219,29 @@ export function addModel(dbAdapter) {
       this.fileExtension = supportedImageTypes[this.mimeType]
       this.noThumbnail = '1' // this may be overriden below
 
-      // Store original image size
       let originalImage = promisifyAll(gm(tmpAttachmentFile))
```

Comment:
Second one is fo re-reading properly rotated image. But I'll see, maybe it can be done more accurately.